src/api/data/deltaker.ts

```diff
@@ -115,7 +116,8 @@ export const deltakerSchema = z.object({
 	historiskeEndringsmeldinger: z.array( endringsmeldingSchema ).nullable(),
 	adresse: adresseSchema.nullable(),
 	gjeldendeVurderingFraArrangor: vurderingSchema.nullable(),
-	historiskeVurderingerFraArrangor: z.array(vurderingSchema).nullable()
+	historiskeVurderingerFraArrangor: z.array( vurderingSchema ).nullable(),
```

Comment:
Selv om det er litt ville vesten med formattering i disse gamle frontend-appene, så synes jeg mellomrom før/etter paranteser er et litt sært formateringsvalg, og ikke helt i stil med resten. 

Kanskje kan eslintrc oppdateres med `space-in-parens: ["error", "never"]`?

Reply:
Skal jeg ta med den nå? det blir ca 30 filer ca med endring, men går fint for min del å ta med :)

Reply:
Aiai, hvis du har tenkt å gjøre store endringer ellers så er det kanskje best å ta den til slutt i allefall 😅